did/helper.go

```diff
@@ -1,51 +1,71 @@
 package did
 
 import (
+	"crypto"
 	"crypto/ecdsa"
 	"crypto/ed25519"
 	"crypto/elliptic"
+	"crypto/x509"
 	"encoding/base64"
 	"encoding/json"
+	"encoding/pem"
 	"errors"
+	"fmt"
+	"net/url"
 	"os"
+	"strings"
 
 	"github.com/lestrrat-go/jwx/v3/jwk"
 	"github.com/trustbloc/kms-go/doc/util/fingerprint"
 	"go.uber.org/zap"
 	"software.sslmate.com/src/go-pkcs12"
 )
 
-func GetDIDKeyFromECPKCS12(path, password, keyType string) (did string, err error) {
-
-	bytes, err := os.ReadFile(path)
-
-	if err != nil {
-		zap.L().Sugar().Warnf("Was not able to read the file %s", path, "error", err)
-		return did, err
+func LoadCertificates(config *Config) (err error) {
+
+	if config.KeyPath != "" || config.CertPath != "" {
```

Comment:
This is quite some deep nesting:) could you extract the config.KeyPath != "" || config.CertPath != "" case to a method, to lower the nesting?